File: syncer/model/rating.js
const Sequelize = require('sequelize');
const sequelize = require('../config/pg')
var Rating = sequelize.define("rating", {
    id: {
        type: Sequelize.BIGINT,
        primaryKey: true,
    },
    productId: {
        type: Sequelize.BIGINT,
        field: 'product_id'
    },
    star: {
        type: Sequelize.INTEGER,
    },
    content: {
        type: Sequelize.TEXT,
    },
    stake: {
        type: Sequelize.INTEGER,
    },
    userId: {
        type: Sequelize.BIGINT,
        field: 'user_id'
    },
    transactionId: {
        type: Sequelize.TEXT,
    },
})

module.exports = Rating

File: BE/org_service/model/rating.go
package model

type Rating struct {
	ID        string `json:"id,omitempty"`
	ProductID string `json:"product_id,omitempty"`
	UserID    string `json:"user_id,omitempty"`
	Star      int    `json:"star,omitempty"`
	Stake     int    `json:"stake,omitempty"`
	Content   string `json:"content,omitempty"`
}
